libraries/AC_Avoidance/AC_Avoidance_Logging.cpp

```diff
@@ -46,6 +47,23 @@ void AP_OADijkstra::Write_OADijkstra(const uint8_t state, const uint8_t error_id
     AP::logger().WriteBlock(&pkt, sizeof(pkt));
 }
 
+void AP_OART_AStar::Write_RTAstar(const AP_OART_AStar::AP_OART_Astar_State& state, const AP_OART_AStar::AP_OA_RTAStar_Error& error_id, const Location &next_loc, uint64_t nodes_time, uint64_t visgraph_time, uint64_t path_time, uint64_t total_time)
+{
+    const struct log_OARTAstar pkt{
+        LOG_PACKET_HEADER_INIT(LOG_OA_RTASTAR_MSG),
+        time_us        : AP_HAL::micros64(),
+        state          : static_cast<uint8_t>(state),
```

Comment:
nitpick: i wonder if these casts are really required?  personally I prefer the simpler "(uint8_t)state" method but it's up to you of course.

Reply:
@rmackay9 you're showing your age :) That method is so 1970's